File: _challenges/6696c2ca763f0712c198dbfa_challenge.js
// NAMEEN:Bacterial growth
// NAMERU:Рост количества бактерий

/*
INSTRUCTIONENSTART
A scientific laboratory is studying the behavior of a certain species of bacteria.
Every day their number is doubled.

Create a variable `bacteriaCount` and give it a value of `10000`.

Then write a command that doubles the existing value of the `bacteriaCount` variable.
INSTRUCTIONENEND
*/

/*
INSTRUCTIONRUSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-10.png)

В научной лаборатории изучают поведение некоторого вида бактерий.
Каждые сутки их количество увеличивается в два раза.

Создайте переменную `bacteriaCount` и присвойте ей значение `10000`.

Напишите следующую команду, которая увеличивает имеющееся значение переменной `bacteriaCount` в два раза.
INSTRUCTIONRUEND
*/

/*
HINTENSTART
```javascript
let bacteriaCount = 10000;
bacteriaCount = bacteriaCount * 2;
```
HINTENEND
*/

/*
HINTRUSTART
```javascript
let bacteriaCount = 10000;
bacteriaCount = bacteriaCount * 2;
```
HINTRUEND
*/

/*
INITIALSOLUTIONSTART

INITIALSOLUTIONEND
*/

// SOLUTIONSTART
let bacteriaCount = 10000;
bacteriaCount = bacteriaCount * 2;
// SOLUTIONEND


// OPENTESTSSTART
it("The `bacteriaCount` variable is not undefined", () => {
    expect(bacteriaCount).not.undefined;
});

it("The `bacteriaCount` variable contains a value 20000", () => {
    expect(bacteriaCount).eq(20000);
});

it("The `bacteriaCount` is type of number", () => {
  expect(typeof bacteriaCount).eq('number');
});
// OPENTESTSEND

// HIDDENTESTSSTART

// HIDDENTESTSEND

File: _challenges/6696c2ca763f0712c198dbfa_challenge.py
// NAMEEN:How many mushrooms are there?
// NAMERU:Сколько стало грибов?

/*
INSTRUCTIONENSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-13.png)

Masha collected `mushroom_count` of mushrooms in the forest. Returning home, she found another 15 mushrooms.

Suppose that the variable `mushroom_count` stores some number.

Create a variable `mushroom_count` with a value of `22:

Write a command that increments the existing value of the variable `mushroom_count` by 15.
INSTRUCTIONENEND
*/

/*
INSTRUCTIONRUSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-13.png)

Маша собрала в лесу `mushroom_count` грибов. Возвращаясь домой, она нашла еще 15 грибов.

Создайте переменную `mushroom_count` со значением `22`.

Запишите команду, которая увеличивает имеющееся значение переменной `mushroom_count` на `15`.
INSTRUCTIONRUEND
*/

/*
HINTENSTART
Similar example:
```python
books = 5
books = books + 9
```
HINTENEND
*/

/*
HINTRUSTART
Похожий пример:
```python
books = 5
books = books + 9
```
HINTRUEND
*/

/*
INITIALSOLUTIONSTART

INITIALSOLUTIONEND
*/

// SOLUTIONSTART
mushroom_count = 22
mushroom_count = mushroom_count + 15
// SOLUTIONEND

// OPENTESTSSTART
it('The "mushroom_count" variable should be defined', () => {
    expect(mushroom_count).not.undefined;
});

it('The "mushroom_count" variable should be equal to 37', () => {
    expect(mushroom_count).to.equal(37);
});

it('The "mushroom_count" variable should be a number', () => {
    expect(mushroom_count).to.be.a('number');
});
// OPENTESTSEND

// HIDDENTESTSSTART

// HIDDENTESTSEND
